libs/api/domains/license-service/src/lib/dto/licenses/LicenseFallbackData.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+/**
+ * Fallback data for license verification
+ * Old verification methods will return JSON data as a string
+ */
+@ObjectType('LicenseFallbackData')
+export class LicenseFallbackData {
+  @Field(() => String, { nullable: true })
+  value?: string
+}
```

Comment:
Should not return a bad schema in the new API and create more legacy client code.